org.zrythm.Zrythm.json

```diff
@@ -0,0 +1,442 @@
+{
+    "app-id": "org.zrythm.Zrythm",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "command": "zrythm_launch",
+    "rename-icon": "zrythm",
+    "copy-icon": true,
+    "finish-args": [
+        "--share=ipc",
+        "--device=dri",
+        "--socket=fallback-x11",
+        "--socket=wayland",
```

Comment:
In general we avoid wayland for plugins as a lot of them don't support it. With VST crashing. Did you test any plugins?

Reply:
reverted to x11 only, indeed some plugins don't work (and carla is apparently buggy too)